Dockerfile

```diff
@@ -56,6 +46,7 @@ LABEL name="MongoDB Atlas Operator" \
 WORKDIR /
 COPY --from=builder /workspace/bin/manager .
 COPY hack/licenses licenses
+COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
```

Comment:
The micro image doesn't include CA certs (nor a package manager to retrieve them) so we get them from the builder. Both RHEL (UBI) and Debian (golang) use Mozilla's, so it shouldn't make a difference.